src/main/java/me/contaria/standardsettings/StandardSettingsConfig.java

```diff
@@ -217,9 +217,13 @@ public boolean hasWidget() {
         this.register("pieDirectory", "f3", StandardGameOptions::getPieDirectory, StandardGameOptions::setPieDirectory, option -> new LiteralText(option.get()), option -> {
             TextFieldWidget widget = new TextFieldWidget(MinecraftClient.getInstance().textRenderer, 0, 0, 120, 20, option.getName());
             widget.setMaxLength(128);
-            widget.setTextPredicate(string -> string != null && string.startsWith("root"));
+            widget.setTextPredicate(string -> string != null && (string.startsWith("root") || string.isEmpty()));
             widget.setText(option.get());
             widget.setChangedListener(string -> {
+                if (string.isEmpty()) {
+                    widget.setText("root");
+                    return;
```

Comment:
Returning here will prevent the option and suggestions from being updated, should probably just be replaced with this:
```suggestion
                    string = "root";
```

Reply:
doing setText calls the listener again

Reply:
Oh wait I`m silly it recalls it so it`s called again, my suggestion would cause it to not update, ignore what i said